statics/js/components/graph-layout.js

```diff
@@ -609,6 +611,14 @@ TopologyGraphLayout.prototype = {
       .attr("xlink:href", this.managerImg(d));
   },
 
+  isNeutronNode: function(d) {
```

Comment:
yeah I thought we could use it later but maybe overkill for such comparison, will remove it